source/load-balancers/tests/sdam-error-handling.yml

```diff
@@ -65,6 +65,10 @@ initialData:
 
 tests:
   - description: only connections for a specific serviceId are closed when pools are cleared
+    runOnRequirements:
+    # This test assumes that two sequential connections receive different serviceIDs.
+    # Sequential connections to a serverless instance may receive the same serviceID.
+    - serverless: forbid
```

Comment:
For posterity, this test had inconsistently failed on the Go driver POC.

This test assumes that two sequential connections get different service IDs.

Serverless currently has multiple load balancers for a given instance. A client may not connect to the same load balancer on each connection. Therefore, we cannot guarantee that two sequential connections get different service IDs.

See [this slack conversation](https://mongodb.slack.com/archives/G01304NFS3C/p1630621416055700).